examples/hooks/components/HitsPerPage.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useCallback } from 'react';
+import { useHitsPerPage, UseHitsPerPageProps } from 'react-instantsearch-hooks';
+import { cx } from '../cx';
+
+export type HitsPerPageProps = React.ComponentProps<'div'> &
+  UseHitsPerPageProps;
+
+export function HitsPerPage(props: HitsPerPageProps) {
+  const { items, refine } = useHitsPerPage(props);
+  const { value: currentValue } =
+    items.find(({ isRefined }) => isRefined) || {};
+
+  const handleChange = useCallback(
+    (event) => {
+      refine(event.target.value);
+    },
+    [refine]
+  );
+  return (
+    <div className={cx('ais-HitsPerPage', props.className)}>
+      <select
+        className="ais-HitsPerPage-select"
+        onChange={handleChange}
+        value={`${currentValue}`}
+      >
+        {items.map((item) => (
+          <option
+            key={item.label + item.value}
```

Comment:
Why do we need the label in the key here? React keys are scoped to their `map`.

Reply:
i picked this up looking at [the Selector component from InstantSearch.js](https://github.com/algolia/instantsearch.js/blob/master/src/components/Selector/Selector.tsx#L39). 
I think it's possible to have twice the same value in items (though the isn't a use case for is here).
If we agree we can remove it I guess.

Reply:
removed in https://github.com/algolia/react-instantsearch/pull/3205/commits/ebb0c124098660e61bd1293d1d145f93ed7a94dc